staticd/static_vrf.c

```diff
@@ -30,33 +30,54 @@
 #include "static_zebra.h"
 #include "static_vty.h"
 
+DEFINE_MTYPE(STATIC, STATIC_RTABLE_INFO, "Static Route Table Info");
```

Comment:
Once again, please use `DEFINE_MTYPE_STATIC` since `MTYPE_STATIC_RTABLE_INFO` is only used on this file.

More details here: http://docs.frrouting.org/projects/dev-guide/en/latest/memtypes.html

Also, maybe you could rename this memtype to `STATIC_VRF` since it's used to hold `static_vrf` structures.

Reply:
done